packages/trader/src/App/Components/Form/__tests__/number-selector.spec.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import NumberSelector from '../number-selector';
+
+const tests_arr_arr_numbers = [
+    [1, 2, 3],
+    [4, 5, 6],
+];
+const mock_props = {
+    arr_arr_numbers: tests_arr_arr_numbers,
+    name: 'test name',
+    onChange: jest.fn(),
+};
+
+describe('NumberSelector', () => {
+    afterEach(() => {
```

Comment:
There was a need to clear mocks, because function info (was it called or not) wasn't cleared after each test case.